public/content/translations/hi/developers/docs/consensus-mechanisms/pow/mining/mining-algorithms/ethash/index.md

```diff
@@ -5,7 +5,7 @@ lang: hi
 ---
 
 <InfoBanner emoji=":wave:">
-   एथाश एथेरियम का प्रूफ-ऑफ-वर्क माइनिंग एल्गोरिथम था। प्रूफ-ऑफ-वर्क को अब **पूरी तरह से बंद कर दिया गया है** और एथेरियम अब इसके बजाय <a href="/developers/docs/consensus-mechanisms/pos/">प्रूफ-ऑफ-स्टेक</a> का उपयोग करके सुरक्षित है। <a href="/roadmap/merge/">द मर्ज</a>, <a href="/developers/docs/consensus-mechanisms/pos/">हिस्सेदारी के सबूत</a> और <a href="/staking/">स्टेकिंग</a> पर अधिक। यह पृष्ठ ऐतिहासिक रुचि के लिए है!  
```

Comment:
@corwintines I've resolved all the issues in the PR, all simple cases of improper handling of tags, apart from this one

In this case, the source string doesn't have tags, do you recall this being an issue in any other files or languages in the past?

<img width="636" alt="Screenshot 2025-01-09 at 13 35 19" src="https://github.com/user-attachments/assets/8d69d17d-57ce-44a4-8fa3-24139ddccbee" />